tools/scripts/run_code_generation.py

```diff
@@ -456,6 +535,15 @@ def main():
 
     highly_refined_percent_of_cores_to_take = 0.9
     max_workers = max(1, int(highly_refined_percent_of_cores_to_take * os.cpu_count()))
+    
+    #build reverse map
+    # Open a file using 'with' and read its contents
```

Comment:
I can get this information from the code.
Please put why this here in comments instead.